dashboards/lodestar_block_processor.json

```diff
@@ -5101,7 +5101,7 @@
             "uid": "${DS_PROMETHEUS}"
           },
           "editorMode": "code",
-          "expr": "rate(lodestar_bls_thread_pool_signature_deserialization_main_thread_time_seconds_bucket[$rate_interval])",
```

Comment:
this one and the below is not right, should be replaced with `lodestar_bls_thread_pool_aggregate_with_randomness_main_thread_time_seconds`

Reply:
nvm the old code was not right